template_feed/Microsoft.DotNet.Common.ProjectTemplates.7.0/content/ClassLibrary-CSharp/.template.config/ide.host.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json.schemastore.org/vs-2017.3.host",
+  "icon": "ide/ClassLibraryCS.ico",
+  "order": 20,
+  "tags": [
+    {
+      "type": "projectType",
+      "add": [],
+      "remove": [ "Common" ]
+    }
+  ],
+  "symbolInfo": [
+    {
+      "id": "langVersion",
+      "isVisible": "true"
```

Comment:
@baronfel this field is text field. Do we need to enable it for VS?

Reply:
Comments below, I'd not. If we want control over this, we should add a choice parameter with the valid values.